app/MMB/src/components/Invoice.js

```diff
@@ -26,7 +26,8 @@ export default class Invoice extends React.Component<{||}> {
       <PdfViewer
         source={{
           uri,
-          cache: true, // no expiration - store as long as possible
+          cache: true,
+          expiration: Infinity, // no expiration - store as long as possible
         }}
```

Comment:
Awesome. So intuitive. Could you please add these two options into `PdfViewer` directly? Let's assume for now that every PDF is going to be cached forever by default. Does it make sense? I don't want to repeat this everywhere because this is not the only place where we use it...